.github/workflows/connector-test-command.yml

```diff
@@ -111,27 +120,24 @@ jobs:
 
       - name: Install connector dependencies
         if: matrix.connector
-        run: |
-          poe connector ${{ matrix.connector }} install
+        working-directory: airbyte-integrations/connectors/${{ matrix.connector }}
+        run: poe install
```

Comment:
This pattern is repeated in several places:

1. Use `working-directory` to make sure we are in the connector's context.
2. Call the simple form of the poe command (`poe install`, `poe test-unit-tests`, etc.)

This is necessary (for now at least) to make this work for manifest-only connectors. However, I think it feels cleaner anyway personally, since the shorter poe invocation is easier to read/scan.